src/Honeybee_Masses2Zones.py

```diff
@@ -85,22 +85,23 @@ def main(maximumRoofAngle, zoneMasses, zoneNames, zonePrograms, isConditioned):
         w = gh.GH_RuntimeMessageLevel.Warning
         ghenv.Component.AddRuntimeMessage(w, warning)
         return -1
-            
+
     # import the classes
     # don't customize this part
     hb_EPZone = sc.sticky["honeybee_EPZone"]
     hb_EPSrf = sc.sticky["honeybee_EPSurface"]
     hb_EPZoneSurface = sc.sticky["honeybee_EPZoneSurface"]
-    
+
     #Have a function to duplicate data.
     def duplicateData(data, calcLength):
         dupData = []
         for count in range(calcLength):
             dupData.append(data[0])
         return dupData
-    
+
     zoneNumber = len(zoneMasses)
-    
+    zoneMasses = map(lambda zm: copy.deepcopy(zm), zoneMasses) # Duplicate the breps so that we don't have same memory access
```

Comment:
@saeranv why do you use lambda? If you want to duplicate each item separately you can do something like this:
```python
zoneMasses = [copy.deepcopy(zm) for zm in zoneMasses]
```
BTW doesn't `zoneMasses = copy.deepcopy(zoneMasses)` work?